Podfile.lock

```diff
@@ -58,6 +58,14 @@ DEPENDENCIES:
   - SQLite.swift (~> 0.11.4)
   - ZXingObjC (~> 3.2.1)
 
+SPEC REPOS:
```

Comment:
@winniequinn What is a spec repo?

Reply:
Ah, don't worry about that. This is purely a work-in-progress PR so I can link to it from elsewhere. I will rebase and remove the changes to this file before submitting it for review.